assets/src/edit-story/components/canvas/layout.js

```diff
@@ -24,12 +24,21 @@ import styled from 'styled-components';
  */
 import { PAGE_NAV_WIDTH, PAGE_WIDTH, PAGE_HEIGHT, HEADER_HEIGHT } from '../../constants';
 import PointerEventsCss from '../../utils/pointerEventsCss';
+import useResizeEffect from '../../utils/useResizeEffect';
+import useCanvas from './useCanvas';
 
 /**
  * @file See https://user-images.githubusercontent.com/726049/72654503-bfffe780-3944-11ea-912c-fc54d68b6100.png
  * for the layering details.
  */
 
+const MENU_HEIGHT = 48;
+const MIN_CAROUSEL_HEIGHT = 65;
+const ALLOWED_PAGE_SIZES = [
+	[ PAGE_WIDTH, PAGE_HEIGHT ],
+	[ 268, 476 ],
```

Comment:
Should these also be constants?

Reply:
They are. But they are not used anywhere independently, so I figured no value in given them names yet?

Reply:
I was thinking of local constants with instead of numbers for readability, def no need to move to the actual `constants` file.

Reply:
The challenge here is that there are no good names here. E.g. should it be `PAGE_WIDTH_2`? Or `PAGE_WIDTH_268`?

Reply:
Based on what did these numbers come from? Could it be for a small screen or in this context min default page width? To reflect the reason for these numbers in the name. Thoughts?

Reply:
I updated it to constants. But I didn't find much better names than `MEDIUM` and `SMALL`. The numbers come from Figma and they are otherwise relative. E.g. we might add another `MEDIUM2` in the mix later or `EVEN_SMALLER` :)

Reply:
I'd vote for `LARGE_SMALL`, `SMALL_LARGE` just for fun (context: https://twitter.com/SheriffAlert/status/1221881862244749315)